zerver/tests/test_bugdown.py

```diff
@@ -937,6 +939,37 @@ def test_mit_rendering(self):
             '<p><a href="https://lists.debian.org/debian-ctte/2014/02/msg00173.html" target="_blank" title="https://lists.debian.org/debian-ctte/2014/02/msg00173.html">https://lists.debian.org/debian-ctte/2014/02/msg00173.html</a></p>',
         )
 
+    def test_slash_commands_trigger(self):
+        # type: () -> None
+        user_profile = self.example_user('hamlet')
+        self.login(user_profile.email)
+        do_change_is_admin(user_profile, True)
+
+        bot_info = {
+            'full_name': 'Sample user1',
+            'short_name': 'user1',
+            'bot_type': UserProfile.SLASH_COMMANDS,
+        }
+        result = self.client_post("/json/bots", bot_info)
+        self.assert_json_success(result)
+
+        msg = Message(sender=user_profile, sending_client=get_client("test"))
+
+        def render(msg, content):
+            # type: (Message, Text) -> Text
+            return render_markdown(msg,
+                                   content,
+                                   message_users={user_profile})
+
+        content = "We have an /user1 day today!"
```

Comment:
Do you want to trigger a slash command in the middle of a message, or only at the beginning?